### ISSUE TITLE
Support draft pull requests

### ISSUE THREAD

**Issue by Serranya** [zerocracy/farm#1909]
Github recently introduced draft pull requests: https://github.blog/2019-02-14-introducing-draft-pull-requests/. This allows to create PRs in a draft status. Indicating that they are not ready for review yet.

Zerocrat should detect draft PRs and should *not* assign a reviewer for these PRs yet.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@serranya[/z](https://www.0crat.com/u/serranya) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by paulodamaso** [comment]
@Serranya @g4s8 I agree.  The problem is that in this state the pull request is not ready for review yet and cannnot be merged. As the code is in draft state, any review made may be wasted, making the reviewer work for free, and worse, the reviewer may lose the job if the performed takes too much time to finish the pull request (time is already ticking for the reviewer after all).

**Comment by paulodamaso** [comment]
@g4s8 Depends on jcabi-github API to support this, I've already made a PR https://github.com/jcabi/jcabi-github/issues/1526
